diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6b5ed8d..d633d49 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -82,6 +82,9 @@ include(src.cmake)
 option(BUILD_CGAME "Build client-side gamelogic" 1)
 option(BUILD_SGAME "Build server-side gamelogic" 1)
 
+include(CMakePrintHelpers)
+cmake_print_variables(RECASTLIST)
+
 if (BUILD_CGAME AND (BUILD_GAME_NATIVE_DLL OR BUILD_GAME_NATIVE_EXE OR NACL))
     if (NACL)
         set(FREETYPE_LIBRARY ${DEPS_DIR}/pnacl_deps/lib/libfreetype.a)
@@ -139,7 +142,7 @@ endif()
 
 add_definitions( -DGLM_FORCE_EXPLICIT_CTOR )
 
-add_library(srclibs-tinygettext EXCLUDE_FROM_ALL ${TINYGETTEXTLIST})
+add_library(srclibs-tinygettext SHARED EXCLUDE_FROM_ALL ${TINYGETTEXTLIST})
 set_target_properties(srclibs-tinygettext PROPERTIES POSITION_INDEPENDENT_CODE 1 FOLDER "libs")
 target_include_directories(srclibs-tinygettext PUBLIC ${LIB_DIR}/tinygettext/include)
 target_compile_definitions(srclibs-tinygettext PUBLIC TINYGETTEXT_UTF8_ONLY)
